bleak/__init__.py

```diff
@@ -412,7 +436,7 @@ async def pair(self, *args, **kwargs) -> bool:
 
     async def unpair(self) -> bool:
         """
-        Unpair with the peripheral.
+        Unpair from the specified GATT server.
```

Comment:
Some additional info to add: Currently this is only available on Windows. Unpairing also disconnects the device if it currently connected.

Reply:
```suggestion
        Unpair from the specified GATT server.

        Unpairing will also disconnect the device.

        This method is only available on Windows and Linux and will raise an
        exception on other platforms.
```